electron/angular-electron/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
```

Comment:
I think whole module electron here is not needed

Reply:
Removed and updated.